src/app/pages/home-page/offer-list/sort/sort.component.scss

```diff
@@ -1,12 +1,10 @@
 .sort-panel{
     background-color: #fff;
-    border-color: rgb(27, 30, 202);
-    border-width: 1em;
     height: 4.5em;
-    width: 100%;
+    width: 99.5%;
```

Comment:
Try to avoid setting sizes in such way. Must be 100%. If you need margin or padding - use it. Recommend to fix next time.

Reply:
I just experiment with div, I forgot to change this into the default value

Reply:
I've already solved this problem